app/helpers/facets_helper.rb

```diff
@@ -0,0 +1,27 @@
+module FacetsHelper
+  # override host's tabs to add all facets as tabs
+  def host_additional_views(host)
+    base_tabs = load_facets(host)
+    base_tabs.merge!(helper_tabs(host))
+    base_tabs
+  end
+
+  def load_facets(host)
```

Comment:
This is not being called from outside `host_additional_views`, might be better if it's private